GitSearch/Helper/DateFormatter+.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  DateFormatter.swift
+//  GitSearch
+//
+//  Created by Vldmr on 12/24/19.
+//  Copyright © 2019 mudritskiy. All rights reserved.
+//
+
+import Foundation
+
+extension DateFormatter {
+    
+    static let localeFormat = NSLocalizedString("date-formatter.locale", tableName: nil, bundle: .main, value: "en_US_POSIX", comment: "Date local display format")
+
+    static let MonthDayYear: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.calendar = Calendar(identifier: .iso8601)
+        formatter.timeZone = TimeZone(secondsFromGMT: 0)
+        formatter.locale = Locale(identifier: DateFormatter.localeFormat)
```

Comment:
`en_US_POSIX ` locale is suitable for parsing dates from backend, but for presenting dates to user its much better to use [current locale](https://developer.apple.com/documentation/foundation/locale/2293654-current) in the system, according to user preferences. See details in Apple's [tech note](https://developer.apple.com/library/archive/qa/qa1480/_index.html).

Reply:
so, when I'm extracting date from query result I can (even need) to use in formatter  **RFC3339DateFormatter** locale as `en_US_POSIX` and in metter to display a date to end-user with formater **MonthDayYear** is good to use `NSLocale.current`? 
Right?

Reply:
Yes, that's right.